open_flamingo/src/helpers.py

```diff
@@ -277,3 +278,187 @@ def forward(
         x = self.ff(x) * self.ff_gate.tanh() + x
 
         return x
+
```

Comment:
This is great! Before we merge, I'll need to test with FSDP; I think this hopefully resolves some of the issues that required freezing the LM embeddings.